liquibase-core/src/main/java/liquibase/datatype/LiquibaseDataType.java

```diff
@@ -198,25 +198,25 @@ public Object sqlToObject(String value, Database database) {
 
     @Override
     public String toString() {
-        String returnString = getName();
+        StringBuilder returnString = new StringBuilder(getName());
         if ((parameters != null) && !parameters.isEmpty() && (maxParameters > 0)) {
-            returnString += "(";
+            returnString.append("(");
             for (Object param : parameters) {
                 if (returnString == null) {
```

Comment:
Same here, `returningString` will never be `null`.

Reply:
Done